public/src/components/channelManagement/bannerTests/bannerTestEditor.tsx

```diff
@@ -114,20 +72,35 @@ const BannerTestEditor: React.FC<BannerTestEditorProps> = ({
     return undefined;
   };
 
-  const updateTest = (updatedTest: BannerTest): void => {
+  const updateTest = (update: (test: BannerTest) => BannerTest): void => {
+    const updatedTest = update(test);
+
```

Comment:
Yeah nice I think it's good to remove the `if` here as it wasn't needed. I think it's maybe slightly unnecessary here for the argument to be a function that updates the test - it might be simpler to just pass in the updated test! For some context, I think the reason it was like this in the `epicTestEditor` was because `test` could potentially be `undefined` so having `updateTest` take an `update` function removed the repetition of having to check `if (test)` in all the `onChange` handlers (`onVariantsChange`, `onRegionsChange`, etc). But now `test` is always defined I think it actually adds a little extra code instead of saving some!

Reply:
Oh! I've just got a little farther and I can see that the `epicTestEditor` `onChange` handlers *did* check it `if(test)` anyways! I really don't know why we have this `update` function then! It's safe to remove though!